### DISCUSSION TITLE
Hosting related

### DISCUSSION THREAD

**Discussion by rahil0405** [tailwindlabs/tailwindcss#9746] [Help]
I have developed my project on tailwind CLI. And it is perfectly working on localhost. I want to deploy it. But whenever I am trying to deploy it it's CSS not working.

**Comment by clondon** [comment]
Hi Rahilya,
You have not given much information here to go off.  But I have a suggestion.
A while back, I experienced similar issues on delopying sites, but the solution was not specific to TailwindCSS.
Mine was related to the baseURL I have set in my project. 
This baseURL changes - this becomes obvious after a while - baseURL changes once you deploy to a server.  You may need to change the baseURL from localhost to that of your deployed project
It might be the situation in your case.
You can check by viewing the source code in your browser and following the link to the CSS file.
